pages/dkp/konvoy/2.1/choose-infrastructure/aks/aks-advanced/aa-self-managed/index.md

```diff
@@ -0,0 +1,105 @@
+layout: layout.pug
```

Comment:
This is the file for the 2.1 documentation. Should this be 2.2 instead?

Reply:
I think because it was a COPS ticket to just add to a section, it somehow got merged with this open PR.  I am verifying if it's ok to push it through as it relates to all self-managed clusters from 2.1 and on into 2.2.  I'll let you know what they tell me.

Reply:
FIxed it!